astrodb_utils/fits.py

```diff
@@ -7,6 +9,7 @@
 
 from astrodb_utils.photometry import assign_ucd
 
+logger = logging.getLogger('astrodb_utils')
```

Comment:
Why not use `logging.getLogger(__name__)` here? It should have the same effect.

Reply:
If I did that, it would be called `astrodb_utils.utils` and it wouldn't be formatted. I don't how to setup ....a child logger which inherits from `astrodb_utils`. But I got this to work for now!  So right now, the plan is to just pass around this one logger with the name `astrodb_utils` as setup in the __init.py__.